### PR TITLE
Rename the general variables to specific meaning

### PR DESCRIPTION
**Pull request by moghadas76**
#### What type of PR is this?

/kind cleanup

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
None
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by yliaog** [comment]
>>    The version of the OpenAPI document: release-1.27
>>    Generated by: https://openapi-generator.tech

the file is generated, they should not be manually edited.

File: kubernetes/client/models/v1_affinity.py [hunk 1]
```diff
@@ -27,12 +27,12 @@ class V1Affinity(object):
 
     """
     Attributes:
-      openapi_types (dict): The key is attribute name
+      openapi_affinity_types (dict): The key is attribute name
```
Comment (roycaihw) [inline kubernetes/client/models/v1_affinity.py:30]: These files are generated from openapi-generator. Please make the change in the upstream generator.

**Comment by Ananya2001-an** [comment]
@roycaihw this PR should be closed ig

**Pull request closed without merging**